app/api/users.py

```diff
@@ -0,0 +1,95 @@
+from flask import Flask
+from flask.ext.restful import Api, Resource, fields, marshal, reqparse
+from app import app
+
+api = Api(app)
+
+user_list = [
+	{
+		'id': 5743, 
+		'first_name': "Scott", 
+		'last_name': "Gladstone",
+		'screenname': "lhpglad", 
+		'email': "happy@rock.com",  
+		'location': "New York",
+		'description': ""
+	},
+	{
+		'id': 5744, 
+		'first_name': "Alex", 
+		'last_name': "Gerstein",
+		'screenname': "algae", 
+		'email': "alex@shnoodles.com",  
+		'location': "New York",
+		'description': ""
+	}
+]
+
+user_fields = {
+	'first_name': fields.String,
+	'last_name': fields.String,
+	'screenname': fields.String,
+	'email': fields.String,
+	'location': fields.String(default = ""),
+	'description': fields.String(default = ""),
+	# 'links': fields.Nested({
+	# 	'feeds': fields.Url('/api/users/{id}/feeds', absolute=True)
+	# 	}),
+	'uri': fields.Url('user')
+}
+
+class UserListAPI(Resource):
+	def __init__(self):
+		self.reqparse = reqparse.RequestParser()
+		self.reqparse.add_argument('first_name', type = str, required = True)
+		self.reqparse.add_argument('last_name', type = str, required = True)
+		self.reqparse.add_argument('screenname', type = str, required = True)
```

Comment:
I think we should just use email as username/screename. An RSS reader doesn't have many public aspects so we don't need this or a description. We arguably don't even need first and last name either.